### ISSUE TITLE
Any plans for adding gen 2 info

### ISSUE THREAD

**Issue by thesquaremedia** [yayramen/godex#9]
Does the library include I for for gen 2 or any plans on adding it?

**Comment by thesquaremedia** [comment]
hey id like to help add gen 2 as well as add images and weaknesses for pokemon is there anything i should keep in mind when editing the data/pokemon.js file?

**Comment by yayramen** [comment]
Weaknesses are already added via types.js and you can use external URLs for images, images in a framework can slow it down. I might be tentatively interested in rereleasing this in the nearing future.

**Comment by thesquaremedia** [comment]
How do you call the weaknesses of the Pokémon? The images is fine I can take care of that through code just though it be easier I added a pull request but I can cancel it and remove the weaknesses and image parameters.

**Comment by yayramen** [comment]
It's just returned in the Pokemon object.

**Comment by thesquaremedia** [comment]
Hmm must’ve missed it somehow I’ll check again thanks.

**Comment by yayramen** [comment]
Check core/Pokemon.js I believe it was added as resistance. So much has changed since I've updated this.

**Comment by thesquaremedia** [comment]
Yea it’s a function of the Pokémon object eg pokemon.resistance(); 

Cool hey so if you want I can help out adding the missing Pokémon and some of the new moves.

**Comment by 12px** [comment]
(This is my new account, still the same guy I promise) If you wanted to add the second gen in the same format as the first gen and update moves, I can start brainstorming how I'd remake the library.

**Comment by thesquaremedia** [comment]
Lol yea whatever is best I’m not really using the iv check mostly the Pokedex part to get info about a Pokémon on the fly like type, moves, dps. So more so like a database.

**Comment by yayramen** [comment]
Yeah, that's going to be the main focus going forward. I'm going to rewrite the library entirely and then host it, so it will be a forkable library, but I should also be able to make it act as a GitHub powered API as well. The reason I was building the pokemon the way I was was to keep the actual data as small as possible. Rather than listing the weaknesses for each pokemon, I list the types for the pokemon, and then refer to the type weakness chart.

**Comment by thesquaremedia** [comment]
yea that makes sense. the only thing i would keep in mind is that you are calculating the maxCP and i believe the formula changed and i beleive lapras uses a separate formula from the rest based on what i've read on the silphroad. modulizing the library makes sense. then on the weaknesses/strenght and move DPS as long as you account for the different types it makes a lot more sense to calculate it. its really a matter of do i want to create all the logic or just input the data directly which means the pokemon data would have to be huge if you add all the information.

**Comment by 12px** [comment]
Check out the development branch. It generates static JSON files for basically everything.
